### PR TITLE
Add Webpay Modal

### PR DESCRIPTION
**Pull request by gonzunigad**
Add Webpay Modal

### CODE REVIEW

File: src/Webpay/WebpayPlus.php [hunk 1]
```diff
@@ -13,96 +13,17 @@
...
-     */
-    public static $httpClient = null;
     private static $apiKey = Options::DEFAULT_API_KEY;
     private static $commerceCode = Options::DEFAULT_COMMERCE_CODE;
     private static $integrationType = Options::DEFAULT_INTEGRATION_TYPE;
 
-    /**
-     * @return string
-     */
-    public static function getApiKey()
```
Comment (ffiebig) [inline src/Webpay/WebpayPlus.php:42]: I see breaking changes, is this meant for next major version ?

Comment (gonzunigad) [reply src/Webpay/WebpayPlus.php:42]: It is just a refactor. This code is now on "ConfiguresEnvironment" trait. :)

**Comment by gonzunigad** [comment]
I forgot to add more tests. I'm not sure how can I test commit, or refund methods without interacting with the payment form :(

File: tests/TransbankWebpayModalTest.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+class TransbankWebpayModalTest extends \PHPUnit\Framework\TestCase
+{
+    /** @test */
+    public function it_creates_a_modal_transaction()
+    {
+        $response = \Transbank\Webpay\Modal\Transaction::create( 1500, 'BuyOrder1', 'affsafas');
+        $this->assertTrue(true);
```
Comment (gonzunigad) [inline tests/TransbankWebpayModalTest.php:9]: WIP: I should fix this.

**Comment by gonzunigad** [comment]
I added more tests and i think that the chagnges request by @ffiebig are not needed (please, confirm) :)

**Review by ffiebig — APPROVED** [review]

**Pull request merged**